src/toxcore/messenger/packet/mod.rs

```diff
@@ -29,6 +29,8 @@ pub use self::file_data::*;
 pub use self::status_message::*;
 pub use self::file_send_request::*;
 
+pub use crate::toxcore::messenger::conference::packet::{Packet as ConferencePacket, *};
```

Comment:
You don't need `*` here. If you need some packets in that test:

```
    encode_decode_test!(
        packet_conference_request_encode_decode,
        Packet::Conference(ConferencePacket::Invite(Invite::new(1, ConferenceType::Text, ConferenceUID::random())))
    );
```

then you can include them inside `mod tests` section and only that entities that you need. Like use crate::toxcore::messenger::conference::packet::Invite as ConferenceInvite and so on

Reply:
done